lib/foreman_remote_execution_core.rb

```diff
@@ -3,18 +3,26 @@
 module ForemanRemoteExecutionCore
   extend ForemanTasksCore::SettingsLoader
   register_settings([:remote_execution_ssh, :smart_proxy_remote_execution_ssh_core],
-                    :ssh_identity_key_file => '~/.ssh/id_rsa_foreman_proxy',
-                    :ssh_user              => 'root',
-                    :remote_working_dir    => '/var/tmp',
-                    :local_working_dir     => '/var/tmp',
-                    :kerberos_auth         => false)
+                    :ssh_identity_key_file   => '~/.ssh/id_rsa_foreman_proxy',
+                    :ssh_user                => 'root',
+                    :remote_working_dir      => '/var/tmp',
+                    :local_working_dir       => '/var/tmp',
+                    :kerberos_auth           => false,
+                    :async_ssh               => false,
+                    :runner_refresh_interval => 60)
```

Comment:
The `runner_refresh_interval` should be different when using `async_ssh` and when not. With non_async run, I would suggest keeping the 1s interval, so that the life output works as expected.